test/functional/inspec_exec_test.rb

```diff
@@ -33,11 +33,12 @@ def stderr
   end
 
   it "cleanly fails if mixing incompatible resource and transports" do
-    # TODO: I do not know how to test this more directly. It should be possible.
+    # TODO: It should be possible to test this more directly.
     inspec "exec -t aws:// #{profile_path}/incompatible_resource_for_transport.rb"
 
-    _(stdout).must_be_empty
-    _(stderr).must_include "Unsupported resource/backend combination: file / aws. Exiting."
+    _(stderr).must_be_empty
```

Comment:
This test is being changed because the previous version would now be counted as a "failed to load" profile. And that's not the point of this test. It now asserts that the expected error message appears on stdout (as it does as a test failure with the CLI reporter).